pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -388,6 +397,75 @@ class OSRow extends React.Component {
     }
 }
 
+const UnattendedRow = ({ validationFailed, unattendedDisabled, unattendedInstallation, os, onValueChanged }) => {
+    const validationStateRootPassword = validationFailed.rootPassword ? 'error' : undefined;
+    const validationStateUserPassword = validationFailed.userPassword ? 'error' : undefined;
+    let unattendedInstallationCheckbox = (
+        <label className="checkbox-inline">
+            <input id="unattended-installation" type="checkbox"
+                checked={unattendedInstallation}
+                disabled={unattendedDisabled}
+                onChange={e => onValueChanged('unattendedInstallation', e.target.checked)} />
+            {_("Run unattended installation")}
+            <FieldLevelHelp content={_("Perform an unattended install using libosinfo's install script support")} close />
```

Comment:
To be honest I found how you have to click on info icon to get the content quite confusing. Especially since we have OverlayTrigger which gets enabled by hovering over this whole row.
So when I howered over the info icon, I got this helpblock. And I thought that's the content of the info icon and did not realize that the helpblock you can see in the screenshot comes from OverLayTrigger for the whole row. I realized I have to click on the icon only now when I'm reviewing it.
![Screenshot from 2019-09-30 17-27-24](https://user-images.githubusercontent.com/42733240/65893207-e276f080-e3a7-11e9-9468-cdce7cd41c7b.png)

Perhaps we can use PF4 Tooltip which you do not have to click on, or do not use 2 helpblock which are kinda "overlaying each other" in the same field?
WDYT @garrett ?
![Screenshot from 2019-09-30 17-35-52](https://user-images.githubusercontent.com/42733240/65893757-caec3780-e3a8-11e9-82e5-9462238747cf.png)

Reply:
Yeah, I agree that this is confusing.

Both of these popups are "smells" in my opinion. (As in "the leftovers in the fridge smell bad; they should be thrown out" kind of smell.)

What if we drop the tooltips and call the feature "Automatic installation" and only show it when it applies? That should let us drop _both_ tooltips. Right?

Reply:
Showing the option only when available means that it will be always hidden before a user selects an OS. Then when a users selects the OS we will just insert a new field in the modal.
`Automatic Installation` vs `Unattended installation`? I also don't see why the first s batter.
Unattended is commonly used as a term when the user doesn't need to do something interactively.

Reply:
I removed the right info help icon.
I don't agree with the 'Automatic installation' label name though, since 'Unattended' is a general term for referring to actions that don't require user interaction.